doc/movement_and_collide.doc.md

```diff
@@ -119,5 +119,23 @@ for (c=0, c<n_elements, c++) {
   }
 }
 
+if (!isaac.recover) {
+    for (c=0, c<n_enemies, c++) {
+      if (!elements.alive)
+        continue;
+    
+      collision_.p.2.x = enemies[c].x
+      collision_.p.2.y = enemies[c].y
+      collision_.hitbox2 = enemies[c].info and ENEMY_SIZE_MASK
+      a = collision()
+      if (!a){
+          isaac.recover = RECOVERY_TIME
+          isaac.hp -= enemies[c].dmg and DMG_MASK
+          if (isaac.hp < 0)
+            // isaac dies
+      }

```

Comment:
I think it's 
~~~C
if (a) { // if collision
    //do things
    break; // line 189, there is a `jr @@damageDone`
}
~~~